components/script/task_queue.rs

```diff
@@ -237,16 +237,16 @@ impl<T: QueuedTaskConversion> TaskQueue<T> {
                             // Reduce the length of throttles,
                             // but don't add the task to "msg_queue",
                             // and neither increment "taken_task_counter".
-                            throttled_length = throttled_length - 1;
+                            throttled_length - 1;
```

Comment:
This was assigning `throttled_length` to `throttled_length - 1`, but now this statement does nothing.